docs/getting-started/integrations-cdn/react-default-cdn.md

```diff
@@ -38,7 +38,228 @@ You get ready-to-use code tailored to your needs!
 
 This guide assumes you have a React project. You can create a basic React project using [Vite](https://vitejs.dev/). Refer to the [React documentation](https://react.dev/learn/start-a-new-react-project) to learn how to set up a project in the framework.
 
-## Component properties
+### Installing react component from npm
+
+Install the `@ckeditor/ckeditor5-react` package:
+
+```bash
+npm install @ckeditor/ckeditor5-react
+```
+
+The `useCKEditorCloud` hook is responsible for returning information that:
+
+* The editor is still downloading from the CDN with the `status = "loading"`.
+* An error occurred during the download when `status = "error"`. Further information is in the error field.
+* About the editor in the data field and its dependencies when `status = "success"`.
+
+Use the `<CKEditor>` component inside your project. The below example shows how to use it with the open-source plugins.
+
+```js
+import React from "react";
+import { CKEditor, useCKEditorCloud } from "@ckeditor/ckeditor5-react";
+
+const CKEditorDemo = () => {
+	const cloud = useCKEditorCloud( {
+		version: "{@var ckeditor5-version}",
+	} );
+
+	if ( cloud.status === "error" ) {
+		return <div>Error!</div>;
+	}
+
+	if ( cloud.status === "loading" ) {
+		return <div>Loading...</div>;
+	}
+
+	const {
+		ClassicEditor,
+		Bold,
+		Essentials,
+		Italic,
+		Paragraph,
+		Undo,
+	} = cloud.CKEditor;
+
+	return (
+		<CKEditor
+			editor={ ClassicEditor }
+			data={ "<p>Hello world!!!</p>" }
+			config={ {
+				toolbar: {
+					items: [ "undo", "redo", "|", "bold", "italic" ],
+				},
+				plugins: [ Bold, Essentials, Italic, Paragraph, Undo ],
+			} }
+		/>
+	);
+};
+```
+
+To use premium plugins, set the `premium` property to `true` in the `useCKEditorCloud` configuration and provide your license key in the `CKEditor` configuration.
+
+```js
+import React from "react";
+import { CKEditor, useCKEditorCloud } from "@ckeditor/ckeditor5-react";
+
+const CKEditorDemo = () => {
+	const cloud = useCKEditorCloud( {
+		version: "{@var ckeditor5-version}",
+		premium: true,
+	} );
+
+	if ( cloud.status === "error" ) {
+		return <div>Error!</div>;
+	}
+
+	if ( cloud.status === "loading" ) {
+		return <div>Loading...</div>;
+	}
+
+	const {
+		ClassicEditor,
+		Bold,
+		Essentials,
+		Italic,
+		Mention,
+		Paragraph,
+		Undo,
+	} = cloud.CKEditor;
+
+	const { SlashCommand } = cloud.CKEditorPremiumFeatures!;
+
+	return (
+		<CKEditor
+			editor={ ClassicEditor }
+			data={ "<p>Hello world!!!</p>" }
```

Comment:
```suggestion
			data={ "<p>Hello world!</p>" }
```

We don't need to be so passive-aggressive.

Reply:
I would say that the creator of the snippet was more excited. But, yeah, right, hah